impress-org/givewp-next-gen
# Feature: add billing address block

## Description

Depends on https://github.com/impress-org/givewp/pull/6837



This PR adds a billing address block that allows users to collect the donor's address during the donation process.

## Affects



Form Builder

## Visuals



![image](https://github.com/impress-org/givewp-next-gen/assets/16308864/5bb7616f-9cbd-444f-8f0e-a65762cc1daa)

![image](https://github.com/impress-org/givewp-next-gen/assets/16308864/084be756-3fce-4852-b421-3e80f56ac755)


## Testing Instructions



1. Create a new block;
2. Add the new billing fields block to the form;
3. Save the form;
4. Create a new donation using the form;
5. Check the donations page to make sure that the billing address was saved.

## Pre-review Checklist






---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205000286853460
